pkgs/applications/editors/poke/default.nix

```
@@ -19,11 +19,11 @@ let
   isCross = stdenv.hostPlatform != stdenv.buildPlatform;
 in stdenv.mkDerivation rec {
   pname = "poke";
-  version = "1.1";
+  version = "1.2";
 
   src = fetchurl {
     url = "mirror://gnu/${pname}/${pname}-${version}.tar.gz";
-    hash = "sha256-zWjfY8dBtBYLEsvqAvJ8RxWCeUZuNEOTDSU1pFLAatY=";
+    hash = "sha256:0qnz7lc681nxa16rw9n9q4spy9fas6m2adrpb49imh34bgdgh77n";
```

Can you use the SRI format?

It can be generated by the new standalone `nix` tool:

```
sh> nix hash to-sri --type sha256 ${hash in long form}
```
Or even
```
sh> nix hash to-sri sha256:<hash here>
```

In this case, I have found here:
```suggestion
    hash = "sha256-9hz42ltkwBoTWTc3JarRyiV/NcHJJp5NUN0GZBg932I=";
```